src/shell/lisa_shell

```diff
@@ -277,9 +277,12 @@ fi
 
 function _lisa-test-usage {
 cat <<EOF
-Usage: lisa-test [args] FILE[:CLASS]
+Usage: lisa-test [--wload-iterations ITERATIONS] [args] FILE[:CLASS]
```

Comment:
But does it really make sense to run a test multiple times?
Eventually, and that's what we do, we run a workload multiple time and aggregate results to define if THE test pass or not... thus there should not be ambiguity and I'm pretty sure we will unlikely need in the future to run A test multiple times.